lib/rubocop/cop/correctors/percent_literal_corrector.rb

```diff
@@ -94,7 +94,7 @@ def process_lines(node, previous_line_num, base_line_num, source_in_lines)
         begin_line_num = previous_line_num - base_line_num + 1
         end_line_num = node.first_line - base_line_num + 1
         lines = source_in_lines[begin_line_num...end_line_num]
-        "\n" + lines.join("\n").split(node.source).first
+        "\n#{(lines.join("\n").split(node.source).first || '')}"
```

Comment:
Perhaps extract `lines.join("\n").split(node.source).first || ''`, or even `lines.join("\n").split(node.source)`, to a local variable? It’s not immediately obvious what either one is.